lib/src/main/java/com/tubitv/media/fsm/concrete/MoviePlayingState.java

```diff
@@ -113,4 +119,18 @@ private void hideVpaidNShowPlayer(final PlayerUIController controller) {
         }
     }
 
+    private boolean shouldShowSubtitle() {
+        boolean result = false;
```

Comment:
Why is this a variable that needs to be saved.  Shouldn't you just return false at the end instead?